### PR TITLE
Adding ability to override a row.

### PR DESCRIPTION
**Pull request by psachs21**
Allows the user to specify a function that would could override the default row generation. This allows for conditional row information (based on clicks, events, etc)

### CODE REVIEW

File: src/table/table.css [hunk 1]
```diff
@@ -7,4 +7,5 @@
 .row {
   display: flex;
   max-width: 100%;
```
Comment (SeeThruHead) [inline src/table/table.css]: redundant?

File: src/table/table.css [hunk 2]
```diff
@@ -7,4 +7,5 @@
...
+  width: 100%;
```
Comment (SeeThruHead) [inline src/table/table.css]: redundant?

File: infl-components-src/tooltip/index.js [hunk 3]
```diff
@@ -59,7 +59,7 @@ export default class Tooltip extends Component {
   static propTypes = {
     title:             PropTypes.string,
     element:           PropTypes.node.isRequired,
-    position:          PropTypes.oneOf(['top', 'bottom']),
+    position:          PropTypes.string,
```
Comment (SeeThruHead) [inline infl-components-src/tooltip/index.js]: is this supposed to be in this pr?

File: infl-styles/_icon.scss [hunk 4]
```diff
@@ -915,12 +915,22 @@
...
   content: "\E05C"
 }
 
-@mixin ic-coins-old() {
+@mixin ic-refresh() {
   font-family: 'influicons';
   content: "\E05C"
 }
 
+
```
Comment (SeeThruHead) [inline infl-styles/_icon.scss]: is this supposed to be in this pr?

File: src/table/table.css (hunk 2 above)
Comment (psachs21) [inline src/table/table.css]: I thought so too, but it needed it.

File: infl-components-src/tooltip/index.js (hunk 3 above)
Comment (psachs21) [inline infl-components-src/tooltip/index.js]: It removes a stupid error that's not really an error. It could be a different PR, it just seems like a dumb PR.

File: src/table/table.css (hunk 2 above)
Comment (SeeThruHead) [inline src/table/table.css]: i mean i think you can remove max-width now right?

Comment (psachs21) [inline src/table/table.css]: ah lol, yeah probably.

File: infl-components-src/tooltip/index.js (hunk 3 above)
Comment (SeeThruHead) [inline infl-components-src/tooltip/index.js]: i think it's supposed to be top bottom left right no?

Comment (psachs21) [inline infl-components-src/tooltip/index.js]: Yes but since the internal component checks it anyways, do we need to duplicate it?

Comment (SeeThruHead) [inline infl-components-src/tooltip/index.js]: i would just remove it then, unless we want to keep it for self-documentation, in which case is should be [top, bottom, left, right]

File: infl-styles/_icon.scss (hunk 4 above)
Comment (psachs21) [inline infl-styles/_icon.scss]: No,  just a side effect.
